src/authenticator/index.ts

```diff
@@ -60,9 +65,17 @@ export class FronteggAuthenticator {
 
       this.accessToken = '';
       this.accessTokenExpiry = 0;
+
+      if (this.maxRetries > this.retriesCount) {
+        Logger.info(`try [${this.retriesCount}] failed to authenticate with frontegg, trying again`);
+        this.retriesCount += 1;
+        this.refreshAuthentication();
```

Comment:
It's a common practice to wait (a not linear) time between retires, in case there is a temp issue we don't want it to try to fast